play-haskell-server/src/Pages.hs

```diff
@@ -29,10 +30,26 @@ loadTemplate fp = do
     Right templ -> return templ
     Left err -> die (show err)
 
-renderPlayPage :: Mustache.Template -> Maybe ByteString -> ByteString
-renderPlayPage templ mcontents = Enc.encodeUtf8 $
-  Mustache.substituteValue templ $ Mustache.object
-    [(Text.pack "preload", mixinMaybeNull (jsStringEncode . decodeUtf8) mcontents)]
+renderPlayPage :: Mustache.Template -> Maybe Paste -> ByteString
+renderPlayPage templ = \case
+  Just paste -> Enc.encodeUtf8 $ Mustache.substituteValue templ $ pasteToMustacheObject paste
+  Nothing -> Enc.encodeUtf8 $ Mustache.substituteValue templ $ Mustache.object [(Text.pack "preload", toMustache False)]
+
+versionToMustache :: Maybe Version -> Mustache.Value
+versionToMustache = \case
+  Just ( Version v) -> toMustache v
```

Comment:
Superfluous space after `(`.
Also, `versionToMustache = mixinMaybeNull (\(Version v) -> v)` if I'm not mistaken.